FirebaseCore/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Firebase 10.13.0
+- For developers building for visionOS, Xcode beta 5 or later is required.
```

Comment:
Xcode 15 beta 5

Reply:
That's an important distinction, thanks for catching!